config/rbac.yml

```diff
@@ -67,6 +67,48 @@ rules:
 - apiGroups: ["flowcontrol.apiserver.k8s.io"]
   resources: ["prioritylevelconfigurations", "flowschemas"]
   verbs: ["list", "watch"]
+
+---
+kind: ClusterRole
```

Comment:
This will cause a conflict when deploying multiple kapp-controllers as it is currently a Cluster-wide role.

This should be a `Role` with relevant `RoleBindings` per namespace.

Reply:
I think it RBAC to monitor to cluster resoource is needed. The clustorRole can be kept and we need to creat clusterrolebinding for the SA kapp-controller-sa in each namespace